sdks/java/container/java21/option-jamm.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
although same options are added as java17, it seems jamm still not working on Java21:

```
JVM prevents jamm from accessing subgraph - cache sizes may be underestimated
```

support left as next task